site/docs/sql/enums/aws_acm_entity_certificate.certificateStatusEnum.md

```diff
@@ -0,0 +1,36 @@
+---
+id: "aws_acm_entity_certificate.certificateStatusEnum"
+title: "Enumeration: certificate_status"
+sidebar_label: "certificate_status"
+custom_edit_url: null
+---
+
+Different states for a certificate. A valid certificate should be in ISSUED state
+
+## Values
+
+• **EXPIRED** = ``"EXPIRED"``
```

Comment:
There is something weird for the enums, it thinks they are some secrets or something 
<img width="551" alt="image" src="https://user-images.githubusercontent.com/5898639/207075935-6e803b3a-a7a1-4fd9-a6f9-666862eeb8c5.png">